CONTRIBUTING.md

```diff
@@ -113,35 +113,35 @@ Once that is complete just run:
 
 ```
 sh
-npm run test && npm run build
+npm run test && npm run build -- --skip-os-packages
 ```
 
 #### Debugging unit tests
 
 The standard `npm run test` task runs several sub tasks and can take several minutes to complete, making debugging failures pretty painful. In order to ease the pain specialized tasks provide alternate methods for running the tests.
 
 
-`npm run test:quick`  
+`npm run test:quick`
```

Comment:
I have no clue why all of these lines had spaces after them, but my editor removed them on save.